CPAC/resources/configs/pipeline_config_monkey.yml

```diff
@@ -0,0 +1,706 @@
+regANTSinterpolation: Linear
+
+# Possible values: trilinear, sinc, spline
+regFSLinterpolation: trilinear
+
+
+# Choose whether to use the mean of the functional/EPI as the input to functional-to-anatomical registration or one of the volumes from the functional 4D timeseries that you choose.
+func_reg_input :  ['Mean Functional']
+
+
+# Only for when 'Use as Functional-to-Anatomical Registration Input' is set to 'Selected Functional Volume'. Input the index of which volume from the functional 4D timeseries input file you wish to use as the input for functional-to-anatomical registration.
+func_reg_input_volume :  0
+
+
+# Choose which tool to be used in functional masking - AFNI 3dAutoMask or FSL BET.
+functionalMasking :  ['3dAutoMask']
+
+# Register functional images to a standard MNI152 template.
+# This option must be enabled if you wish to calculate any derivatives. If set to On [1], only the template-space files will be output. If set to On/Off [1,0], both template-space and native-space files will be output.
+runRegisterFuncToMNI :  [1]
+
+
+# The resolution (in mm) to which the preprocessed, registered functional timeseries outputs are written into. Note that selecting a 1 mm or 2 mm resolution might substantially increase your RAM needs- these resolutions should be selected with caution. For most cases, 3 mm or 4 mm resolutions are suggested.
+resolution_for_func_preproc :  3mm
+
+
+# The resolution (in mm) to which the registered derivative outputs are written into.
+resolution_for_func_derivative :  3mm
+
+
+# Standard FSL Skull Stripped Template. Used as a reference image for functional registration
+template_brain_only_for_func :  /Users/hecheng.jin/Monkey/NMT/NMTtemplate/NMT_SS_2mm.nii.gz
+
+
+# Standard FSL Anatomical Brain Image with Skull
+template_skull_for_func :  /Users/hecheng.jin/Monkey/NMT/NMTtemplate/NMT_2mm.nii.gz 
+
+
+# Matrix containing all 1's. Used as an identity matrix during registration.
+# It is not necessary to change this path unless you intend to use non-standard MNI registration.
+identityMatrix :  $FSLDIR/etc/flirtsch/ident.mat
+
+
+# Run ICA-AROMA de-noising.
+runICA :  [0]
+
+
+# Types of denoising strategy: i)nonaggr-patial component regression, ii)aggr-aggressive denoising
+aroma_denoise_type :  nonaggr
+
+
+# Run Nuisance Signal Regression
+runNuisance :  [1]
+
+
+# Standard Lateral Ventricles Binary Mask
+# lateral_ventricles_mask :  $FSLDIR/data/atlases/HarvardOxford/HarvardOxford-lateral-ventricles-thr25-2mm.nii.gz
+lateral_ventricles_mask :  /Users/hecheng.jin/Monkey/NMT/NMTtemplate/fullmask.nii.gz
+
+# Select which nuisance signal corrections to apply
+Regressors :
+
+ - Motion:
+     include_delayed: true
+     include_squared: true
+     include_delayed_squared: true
+
+   aCompCor:
+     summary:
+       method: DetrendPC
+       components: 5
+     tissues:
+       - WhiteMatter
+       - CerebrospinalFluid
+     extraction_resolution: 2
+
+#   CerebrospinalFluid:
```

Comment:
Why is this commented out?

Reply:
I didn't realize NMT template has CSF segmentation mask, so I've run without csf at the beginning. It can be uncomment.